### ISSUE TITLE
Add a way to calculate the total burnt loki

### ISSUE THREAD

**Issue by Doy-lee** [loki-project/loki-core#1034]
Similar to how we can calulate the coinbase tx sum (https://github.com/loki-project/loki-core/blob/14f687ef57bc9fee70296c253edf044c85b88013/src/cryptonote_core/cryptonote_core.cpp#L1618) we should be able to do the same with burnt loki.

It might good to just extend coinbase tx sum for that purpose.

**Comment by jagerman** [comment]
Related: https://github.com/loki-project/loki-onion-blockchain-explorer/issues/17 would use this.

**Comment by TakahikoKarasawa** [comment]
I'm trying to resolve this issue. I want to know domain knowladge of Loki burning.
Could you give me some useful resources or links about loki burning.

**Comment by neuroscr** [comment]
See https://github.com/loki-project/loki-onion-blockchain-explorer/pull/19/commits/34cc01a21a0187e6c29a87d71061047c5616bf47

**Comment by Doy-lee** [comment]
Apologies, this issue has been fixed by @darcys22 and the explorer has burn implemented in https://github.com/loki-project/loki-onion-blockchain-explorer/pull/19/commits/34cc01a21a0187e6c29a87d71061047c5616bf47

We don't have documentation at the moment on how burns work, the best would be to look at the code base and the this PR was the one that implemented it https://github.com/loki-project/loki-core/pull/920

In summary, burns are encoded as a TX extra field in atomic units and are added to the TX fee. The miner of a block then subtracts the burn from the cumulative fee (that includes the burn) when it generates a block.

**Comment by TakahikoKarasawa** [comment]
Oh no I didn't know this issue had finished. Thank you for showing links.
To tell the truth, I cannot fully understand loki burning, but I think it must be trivial problem, so I go on next new isses.
Thanks
